docs/content/configuration/notifications/smtp.md

```diff
@@ -186,10 +186,11 @@ You need to generate an app password in order to use Gmail SMTP servers. The pro
 ```yaml {title="configuration.yml"}
 notifier:
   smtp:
+    address: 'smtp://smtp.gmail.com:587'
     username: 'myaccount@gmail.com'
     # Password can also be set using a secret: https://www.authelia.com/configuration/methods/secrets/
     password: 'yourapppassword'
     sender: 'admin@{{< sitevar name="domain" nojs="example.com" >}}'
-    host: 'smtp.gmail.com'
-    port: 587
 ```
+
+[security]: ../../overview/security/measures.md#notifier-security-measures-smtp
```

Comment:
It doesn't look like this renders correctly. 
I don't think that its a good idea to clutter the configuration reference page with miscellaneous information. 
Some people are saying that our site is cluttered with information that makes it hard to understand.